Gems/ScriptCanvas/Code/Include/ScriptCanvas/Grammar/ParsingUtilities.cpp

```diff
@@ -1567,3 +1569,8 @@ namespace ScriptCanvas
         }
     }
 }
+
+#include <ScriptCanvas/Libraries/Time/RepeaterNodeable.h>
+#include <ScriptCanvas/Libraries/Time/DelayNodeable.h>
+#include <ScriptCanvas/Libraries/Time/TimeDelayNodeable.h>
+#include <ScriptCanvas/Libraries/Logic/Gate.h>
```

Comment:
missing newline at the end of this file
```suggestion
#include <ScriptCanvas/Libraries/Logic/Gate.h>

```